mir-cli/pom.xml

```diff
@@ -30,7 +30,12 @@
         <artifactId>appassembler-maven-plugin</artifactId>
         <configuration>
           <!-- Extra JVM arguments that will be included in the bin scripts -->
-          <extraJvmArguments>-DMCR.AppName=${MCR.AppName}</extraJvmArguments>
+          <extraJvmArguments>-DMCR.AppName=${MCR.AppName}
```

Comment:
This needs to go into https://github.com/MyCoRe-Org/mycore-skeleton-archetype
as well, because this is the basis for new mycore applications.